pandas/tests/series/test_indexing.py

```diff
@@ -547,6 +547,29 @@ def test_getitem_setitem_periodindex(self):
         result[4:8] = ts[4:8]
         assert_series_equal(result, ts)
 
+    @pytest.mark.parametrize(
+        'result_1, duplicate_item, expected_1, duplicate_key, unique_key',
+        [
+            [
+                pd.Series({1: 12, 2: [1, 2, 2, 3]}), pd.Series({1: 313}),
+                pd.Series({1: 12, }, dtype=object),
+                1, 2,
+            ],
+            [
+                pd.Series({1: [1, 2, 3], 2: [1, 2, 2, 3]}),
+                pd.Series({1: [1, 2, 3]}), pd.Series({1: [1, 2, 3], }),
+                1, 2,
+            ],
+        ])
+    def test_getitem_with_duplicates_indices(
+            self, result_1, duplicate_item,
```

Comment:
the duplicate_key and unique_key are the same in each parametrization yes? so don't include them, just directly index in the test body